openapi_python_client/templates/model.pyi

```diff
@@ -33,14 +46,17 @@ class {{ model.reference.class_name }}:
         {% endif %}
         {% endfor %}
 
-
-        field_dict = {
+        field_dict: Dict[str, Any] = {}
+        {% if model.additional_properties %}
+        field_dict.update(self._additional_properties)
+        {% endif %}
+        field_dict.update({
             {% for property in model.required_properties + model.optional_properties %}
             {% if property.required %}
             "{{ property.name }}": {{ property.python_name }},
             {% endif %}
             {% endfor %}
-        }
+        })
```

Comment:
Why not leave the regular properties in the field dict declaration? Would be a bit faster I think rather than having to update after the fact, even without having additional properties.